gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-edit-page/api-documentation-v4-edit-page.component.spec.ts

```diff
@@ -508,6 +508,46 @@ describe('ApiDocumentationV4EditPageComponent', () => {
     });
   });
 
+  describe('Delete page', () => {
+    describe('In the root folder', () => {
+      describe('with published page', () => {
+        const PAGE = fakeMarkdown({ id: 'page-id', name: 'page-name', content: 'my content', visibility: 'PUBLIC', published: true });
+        const OTHER_PAGE = fakeMarkdown({
+          id: 'other-page',
+          name: 'other-page-name',
+          content: 'my other content',
+          visibility: 'PUBLIC',
+          published: true,
+        });
+
+        beforeEach(async () => {
+          await init(undefined, PAGE.id);
+          initPageServiceRequests({ pages: [PAGE, OTHER_PAGE], breadcrumb: [], parentId: undefined, mode: 'edit' }, PAGE);
+        });
+
+        it('should delete page and navigate to root list', async () => {
```

Comment:
Could add another test that after deleting a nested page, it goes back to nested parent